src/widgets/vitals/vitals-chart.component.tsx

```diff
@@ -88,27 +88,27 @@ const VitalsChart: React.FC<VitalsChartProps> = ({
   const vitalSigns = [
     {
       id: "bloodPressure",
-      title: `BP (${bloodPressureUnit})`,
+      title: `BP ${bloodPressureUnit ? `(${bloodPressureUnit})` : ""}`,
```

Comment:
You could also use `${bloodPressureUnit ?? ''}`

Reply:
Would `??` work too in the case where if `bloodPressureUnit` is truth, I want it to return the unit encased in brackets and an empty string otherwise?

`BP (mmHg)` if units specified
`BP` otherwise

Reply:
Nope, `??` only works for `undefined` and `false`. But I guess that is exactly what you are after (see screenshot).

Needless to say there is the `()` that I did not notice. So scratch that idea. All good (even though a helper would still make sense).